src/main/resources/application.yml

```diff
@@ -17,6 +17,17 @@ spring:
   data:
     mongodb:
       uri: mongodb://${DB_USER:admin}:${DB_PASSWORD:pwd}@${DB_HOST:localhost}:${DB_PORT:27017}/${DB_NAME:sync}?authSource=${AUTH_SOURCE:admin}
+  redis:
+    host: ${REDIS_HOST:localhost}
+    port: ${REDIS_PORT:6379}
+    user: ${REDIS_USER:default}
+    password: ${REDIS_PASSWORD:password}
+    timeout: 60000
+    database: 0
+    time-to-live: ${REDIS_DATA_TTL:2}
+    requests-cache:
+      database: 1
+      time-to-live: ${REDIS_REQUEST_TTL:4}
```

Comment:
This should probably default to a day or a week.

Reply:
The default to use depends on what we want to do in the case of a failed request.

 1. We want to pick the failure up in the DLQ and manually intervene however needed, no need to allow `tis-trainee-sync` to send a new request because we'll take care of it.
 2. We want to allow new requests for data that was not received a reasonable time after the initial request, automatically assuming that the request hit the DLQ somewhere but might magically work this time.

The middle ground would be to set the timeout to be roughly the same as the DLQ timeout (4 days currently), so if the request expires in the DLQ this service can request it again straight away if needed.